external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -1406,10 +1413,10 @@ impl SymbolicatorRunner {
                         }
                     };
                     if let Some(all_starting_paths) = all_starting_paths_opt {
```

Comment:
I am really confused about the naming of the variables in this code here:

- Why `Some(root_dir)` when that is a set?
- What does `all_starting_paths` _mean_?